src/tests/end-to-end/common/page-controllers/background-page.ts

```diff
@@ -26,14 +26,14 @@ export class BackgroundPage extends Page {
     public async setHighContrastMode(enableHighContrast: boolean): Promise<void> {
         await this.waitForInitialization();
         await this.evaluate(enable => {
-            window.insightsUserConfiguration.setHighContrastMode(enable);
+            void window.insightsUserConfiguration.setHighContrastMode(enable);
```

Comment:
I think it would be preferable not to void this; Playwright's evaluate is Promise-aware and will await the result if an evaluated function returns a promise, I think it would be preferable to let it do so

Reply:
(same for `setTelemetryState`)